src/components/Patient/PatientHome.tsx

```diff
@@ -490,34 +473,36 @@ export const PatientHome = (props: {
                           <CareIcon icon="l-file-upload" className="text-xl" />
                           {t("view_update_patient_files")}
                         </span>
-                      </ButtonV2>
+                      </Button>
                     </div>
 
                     {NonReadOnlyUsers && (
                       <div>
-                        <ButtonV2
+                        <Button
                           id="assign-volunteer"
                           onClick={() => setOpenAssignVolunteerDialog(true)}
                           disabled={false}
+                          variant="outline"
                           authorizeFor={NonReadOnlyUsers}
-                          className="w-full bg-white font-semibold text-green-800 hover:bg-secondary-200"
-                          size="large"
+                          className="w-full text-green-800 font-semibold hover:text-green-900"
+                          size="lg"
```

Comment:
let's stick with colors from variant instead of manually overriding these.

Reply:
In design the button text is green , but in variants we dont have bg-white and green text varient. so i overrided it.

Should i add a new varient for it?

Reply:
@rithviknishad This is how its look like now

![image](https://github.com/user-attachments/assets/88ec383e-106f-456d-9167-48f6115bd24f)

Is this fine